docs/Software Engineering Blog/posts/Comparing_Human_and_AI_way_for_Big-String-Set_Intersection_Performance_in_Java_by_JMH.md

```diff
@@ -0,0 +1,115 @@
+# Comparing Human and AI way for Big-String-Set Intersection Performance in Java by JMH
+
+Recently, I'm facing a case on my job, comparing two set of users and find the intersection as the result. But the difficult part is the number of set is quite big (both set could be 10,000+ elements). To provide a better experiment for our user, the speed can not be too slow. So I want to find a relatively quickly way to do the intersection work. 
```

Comment:
📝 **[vale]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>[write-good.E-Prime] Try to avoid using 'be'.